src/lib/wc-base.js

```diff
@@ -69,7 +69,7 @@ export const baseStyles = css`
 
 	.u-skipLink {
 		position: absolute;
-		top: calc(-100% + -1rem);
+		top: calc(-1000%);
```

Comment:
If skip links are removed from the `cap-nav` element, and are the first thing on every page, I'm not sure this change will be necessary. Did you see an issue before you added the notification banner to the page?

Reply:
Ah, I forgot the map had a skip link — if there is an issue, you can flag it for me and I can fix it in a separate PR. Likely the easiest option would be to reuse the code we use for visually hidden elements (which would hide things in a much more robust way).